code/game/objects/items/weapons/grenades/grenade.dm

```diff
@@ -11,6 +11,7 @@
 	slot_flags = SLOT_BELT
 	var/active = 0
 	var/det_time = 50
+	var/log = TRUE

```

Comment:
This smells like a bad idea. Holocarp grenades have meaningful uses, as do fake grenades (in complaints, specifically). Could make a `var/fake` which interjects a `FAKE` into the attack message.

```dm
msg_admin_attack("[user.name] ([user.ckey]) primed \a [src] [fake ? "(FAKE) " : ""](<A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[user.x];Y=[user.y];Z=[user.z]'>JMP</a>)",ckey=key_name(user))
```